opentelemetry-api/tests/test_implementation.py

```diff
@@ -25,6 +25,8 @@ class TestAPIOnlyImplementation(unittest.TestCase):
     https://github.com/open-telemetry/opentelemetry-python/issues/142
     """
 
+    # TRACER
```

Comment:
What about creating different unit tests for tracer and metrics?

Reply:
I've separated the tests into different files and moved them into the appropriate folders.